qa-tests-backend/src/test/groovy/CSVTest.groovy

```diff
@@ -135,6 +138,18 @@ class CSVTest extends BaseSpecification {
         ${CVE_POSTGRES_FIELDS_FRAGEMENT}
         """
 
+    static final private Map<String, String> QUERIES = [
```

Comment:
nit: have a consistent order between keywords `static`, `private` and `final` across the file. `IMAGE_SHA` does not have the same sequence.

Reply:
Good point. I looked at codenarc options and there is no rule to enforce this :(